<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>base_widget</class>
 <widget class="QWidget" name="base_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>275</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="horizontalSpacing">
      <number>6</number>
     </property>
     <item row="1" column="0">
      <widget class="QCheckBox" name="preset_enabled_cb">
       <property name="text">
        <string>Presets</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="action_label">
       <property name="text">
        <string>Action</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="4">
      <widget class="QComboBox" name="action_box">
       <item>
        <property name="text">
         <string>Ban</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Kick</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="duration_label">
       <property name="text">
        <string>Duration</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="edit_preset_button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="3">
      <widget class="QComboBox" name="selected_preset_box">
       <item>
        <property name="text">
         <string>Spam</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QCheckBox" name="duration_permanent_cb">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Permanent</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="3">
      <widget class="QSpinBox" name="duration_box">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="suffix">
        <string> Hour(s)</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>999999999</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="details_label">
       <property name="text">
        <string>Details</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="details_edit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="button_box">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>preset_enabled_cb</sender>
   <signal>toggled(bool)</signal>
   <receiver>duration_box</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>52</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>preset_enabled_cb</sender>
   <signal>toggled(bool)</signal>
   <receiver>selected_preset_box</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>52</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>preset_enabled_cb</sender>
   <signal>toggled(bool)</signal>
   <receiver>duration_permanent_cb</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>52</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>416</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>duration_permanent_cb</sender>
   <signal>clicked(bool)</signal>
   <receiver>duration_box</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>416</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>preset_enabled_cb</sender>
   <signal>toggled(bool)</signal>
   <receiver>details_edit</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>52</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
